word2vec/word2vec.js

```diff
@@ -22,6 +22,21 @@ class Word2Vec {
     return sum / (this.magnitude(v1) * this.magnitude(v2)); //magnitude is 1 for all feature vectors
   }
 
+  // Add two word vectors
+  static add(v1, v2) {
+    return v1.map((a, i) => a + v2[i]);
```

Comment:
now I have to get my head around `map()`, thank you!!

Reply:
I'm as much as noob as you here :D It's rather simple and practical, really :-)